test/loginjection.test.ts

```diff
@@ -15,32 +15,41 @@
  */
 
 import * as assert from 'assert';
+import * as util from 'util';
 import { Writable } from 'stream';
 import { context, trace } from '@opentelemetry/api';
 import { startTracing, stopTracing } from '../src/tracing';
+import { defaultLogHook } from '../src/instrumentations/logging.ts';
 import type * as pino from 'pino';
 import type * as bunyan from 'bunyan';
 import type * as winston from 'winston';
+import { PinoInstrumentation } from '@opentelemetry/instrumentation-pino';
 
 describe('log injection', () => {
   let stream: Writable;
   let record: any;
 
-  function assertInjection(logger, done, extra?) {
+  function assertInjection(logger, extra?) {
     const span = trace.getTracer('test').startSpan('main');
+    extra = extra ?? [['service.name', 'test-service']];
```

Comment:
`function assertInjection(logger, extra = [['service.name', 'test-service']]) {`

Reply:
Moved the default to signature.